setup.py

```diff
@@ -65,17 +64,19 @@
 here = os.path.abspath(os.path.dirname(__file__))
 
 try:
-    with io.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
-        long_description = '\n' + f.read()
+    with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+        long_description = f"\n{f.read()}"
 except FileNotFoundError:
     long_description = DESCRIPTION
 
-about = {}
 if not VERSION:
+    g = {}
+
+    # TODO: consolidate how we do this
     with open(os.path.join(here, NAME, '__version__.py')) as f:
-        exec(f.read(), about)
-else:
-    about['__version__'] = VERSION
+        exec(f.read(), g)
+        VERSION = g['__version__']
```

Comment:
is this really how to do this? `exec` statements make me anxious

Reply:
I guess it's fine

Reply:
I'm not sure this is the right way to do this, but this is how the existing setup was. I think it makes sense to just put the version tag in this file.